.gitignore

```diff
@@ -7,4 +7,7 @@ dist/*
 .venv
 .env/
 Pipfile
-*.egg-info
\ No newline at end of file
+*.egg-info
```

Comment:
Please fix this line.  It's not secure the way you did it.
The software constructs all or part of an OS command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the ...

📄 [Mitre](https://cwe.mitre.org/index.html)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure